e107_admin/admin.php

```diff
@@ -522,7 +524,8 @@ private function checkIncompatiblePlugins()
 		if($inCompatText)
 		{
 			$text = "<ul>".$inCompatText."</ul>";
-			$mes->addWarning("The following plugins are not compatible with this version of e107 and should be uninstalled: ".$text."<a class='btn btn-default' href='".e_ADMIN."plugin.php'>uninstall</a>");
+			$message = e107::getParser()->lanVars(ADLAN_189, array('x'=>$text, 'y'=>"<a class='btn btn-default' href='".e_ADMIN."plugin.php'>uninstall</a>"));
```

Comment:
No need for using [x] or [y] . 
Just need 2 lans..  `LAN1 . $text . "<a href..... >".LAN2."</a>";`